### ISSUE TITLE
Add arm64 prebuilt packages ?

### ISSUE THREAD

**Issue by abcfy2** [darylounet/libnginx-mod-brotli#3]
Offical nginx repo support arm64 arch: http://nginx.org/packages/ubuntu/pool/nginx/n/nginx/

Would you please also support arm64 packages ?

Thanks.

**Comment by abcfy2** [comment]
I found docker supports `docker buildx` for cross platform building https://docs.docker.com/build/building/multi-platform/ . I think I can create a PR to solve this issue. @darylounet

**Comment by darylounet** [comment]
Hi, sorry for the late answer.
Yeah it is a good idea, but GitHub Actions does not officially support arm64 : https://github.com/actions/runner-images/issues/5631
I've seen workarounds to build using QEMU, but if Github Actions releases arm64 support soon, we'll have to do the job again :-/

**Comment by abcfy2** [comment]
Yeah, that's possible. Because github action has a [docker/setup-qemu-action](https://github.com/docker/setup-qemu-action) to setup qemu environment which required for docker buildx multiple platform.

So I think maybe it's not very hard to resolve this.

EDIT: I succeed use this feature in my repo: https://github.com/abcfy2/docker-postgresql-dev-packages , which can build multiple platform packages successfully.

And if you wish to use docker build instead of docker run, [docker/setup-buildx-action](https://github.com/docker/setup-buildx-action#usage) have this sample.

**Comment by darylounet** [comment]
Resolved with d4d66e7d5626d1daf05c0d7c1ebd6d22d85c62b2
